pkg/virt-handler/vm.go

```diff
@@ -744,6 +745,8 @@ func (d *VirtualMachineController) migrationTargetUpdateVMIStatus(vmi *v1.Virtua
 				return err
 			}
 		}
+
+		vmiCopy.Status.MigrationState.VmToPodIfaceMapping = namescheme.CreateNetworkNameScheme(vmi.Spec.Networks)
```

Comment:
Is it possible to get this information from the migration target pod?

Reply:
I don't follow. You mean reading the pod's links via netlink ? Keep in mind virt-handler doesn't *know* the pod (has no access to the pod's spec / status).